test/environment.js

```diff
@@ -1,8 +1,8 @@
-require('babel-core/register')({
-  ignore: /node_modules/,
+require('@babel/register')({
+  ignore: [ /node_modules/ ],
 })
 
-require('babel-polyfill')
```

Comment:
With the target of node 8, we should be able to get away from using `babel/polyfill` as well